### PR TITLE
MLPAB-2496: Update designs for successful ID matches

### PR DESCRIPTION
**Pull request by acsauk**
# Purpose

Fixes MLPAB-2496

### CODE REVIEW

File: web/template/voucher/one_login_identity_details.gohtml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 {{ define "main" }}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-xl">{{ tr .App "yourIdentityConfirmedWithOneLogin" }}</h1>
+            {{ template "notification-banner" (notificationBanner .App "success" (trHtml .App "youHaveSuccessfullyConfirmedYourIdentitySuccess:voucher") "success" "contents" ) }}
```
Comment (hawx) [inline web/template/voucher/one_login_identity_details.gohtml:10]: the summary list needs updating here too

File: web/template/layout/identity-details.gohtml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{{ define "identityDetails" }}
```
Comment (hawx) [inline web/template/layout/identity-details.gohtml:1]: can this be `identity-details` to match the naming of others

File: web/template/voucher/one_login_identity_details.gohtml (hunk 1 above)
Comment (acsauk) [reply web/template/voucher/one_login_identity_details.gohtml:10]: I'd left this as had assumed we had been requested to not play back DOB and address but if thats not the case I'll update.

File: web/template/donor/onelogin_identity_details.gohtml [hunk 3]
```diff
@@ -4,94 +4,98 @@
...
 
-            {{ template "summary-row" (summaryRow .App "lastName" $userData.LastName "" $donorFullName false true ) }}
+                {{ template "providedDetails" . }}
+                {{ template "identityDetails" . }}
+            {{ else if .DetailsMatch }}
+                {{ template "notification-banner" (notificationBanner .App "success" (trHtml .App "youHaveSuccessfullyConfirmedYourIdentitySuccess:donor") "success" "contents" ) }}
 
-            {{ template "summary-row" (summaryRow .App "dateOfBirth" (formatDate .App $userData.DateOfBirth) "" $donorFullName false true ) }}
+                {{ template "identityDetails" . }}
+            {{ else if not .DetailsMatch }}
```
[the remark this answers is not in the document]
Comment (acsauk) [reply web/template/donor/onelogin_identity_details.gohtml:66]: True, I'd had the orderings different when I was updating the logic and this made more sense at one point.

File: web/template/voucher/one_login_identity_details.gohtml (hunk 1 above)
Comment (hawx) [reply web/template/voucher/one_login_identity_details.gohtml:10]: that is a good question, but I assumed we'd want the same style design even if not all the data

**Review by hawx — APPROVED** [review]

**Pull request merged**